recipes/r-hdrcde/meta.yaml

```diff
@@ -1,94 +1,49 @@
+{% set version = '3.1' %}
```

Comment:
Do you want me to manually make all of these changes in all of these scripts where I add these lines? (example from r-drc I made last week)

```
{% set name = 'drc' %}
{% set version = '3.0-1' %}

package:
  name: r-{{ name|lower }}
  version: {{ version|replace("-", "_") }}

source:
  fn: {{ name }}_{{ version }}.tar.gz
  url:
    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
  md5: 50429c71d876eabdb399a81f1b5913a1


```